### ISSUE TITLE
better default implementation for images

### ISSUE THREAD

**Issue by slemke** [th-koeln/open-product-evaluation-server#216]
The server throws an error if you try to change the default images by uploading / deleting. This seems only to happen with the image preview since it uses the old images code:

```
Uncaught (in promise) Error: GraphQL error: Image deletion failed.
    at new ApolloError (ApolloError.js?d4ec:43)
    at Object.next (QueryManager.js?96af:191)
    at notifySubscription (Observable.js?a7b2:130)
    at onNotify (Observable.js?a7b2:165)
    at SubscriptionObserver.next (Observable.js?a7b2:219)
    at notifySubscription (Observable.js?a7b2:130)
    at onNotify (Observable.js?a7b2:165)
    at SubscriptionObserver.next (Observable.js?a7b2:219)
    at linkFetch.then.then.then.result (index.js?cf67:82)
```

Also see #180. This code should be removed / fixed / improved.

**Comment by ddubbert** [comment]
Default Images get removed completely from the backend. They were only needed for seeding. Clients should now implement their own defaults for like- and dislike icons.

**Comment by slemke** [comment]
implemented in #226
